openfisca_france/parameters/prestations/aides_logement/autres/nr_seuil.yaml

```
@@ -10,3 +10,5 @@ values:
   2017-01-01:
     value: 23.0
     reference: https://www.legifrance.gouv.fr/affichCodeArticle.do;jsessionid=481E3574E9A301CED3336AF168A5EF33.tplgfr44s_1?idArticle=LEGIARTI000035671411&cidTexte=LEGITEXT000006073189&categorieLien=id&dateTexte=
+  2021-01-01:
+    value: 24.0
```

Ajouter la référence législative ou la remonter au niveau supérieur du paramètre si le taux reste invarié à 0,68% ? 
```suggestion
  2021-01-01:
    value: 24.0  # 0.68% du plafond mensuel de sécurité sociale arrondi à l'euro supérieur
    reference: https://www.legifrance.gouv.fr/codes/id/LEGIARTI000035671411/2017-10-01
```